client/web/dev-sandbox/public/static/misc/ftml-test.ftml

```diff
@@ -0,0 +1,258 @@
+//italic text//
+**bold text**
+//**italic and bold**//
+__underline text__
+--strikethrough text--
+{{teletype (monospaced) text}}
+normal ^^superscript^^
+normal ,,subscript,,
+[!-- invisible comment --]
+[[span style="color:red"]]custom //span// element[[/span]]
+[[span id="mySpan"]]My span element[[/span]]
+[[span id="u-mySpan"]]My span element[[/span]]
+##blue|predefined## or ##44FF88|custom-code## color
+
+[[size smaller]]smaller text[[/size]]
+[[size larger]]larger text[[/size]]
+[[size 80%]]80% of current size[[/size]]
+[[size 100%]]100% of current size[[/size]]
+[[size 150%]]150% of current size[[/size]]
+[[size 0.8em]]80% of current size[[/size]]
+[[size 1em]]100% of current size[[/size]]
+[[size 1.5em]]150% of current size[[/size]]
+
+[[size xx-small]]xx-small text[[/size]]
+[[size x-small]]x-small text[[/size]]
+[[size small]]small text[[/size]]
+[[size large]]large text[[/size]]
+[[size x-large]]x-large text[[/size]]
+[[size xx-large]]xx-large text[[/size]]
+[[size 7px]]text size 7 pixels[[/size]]
+[[size 18.75px]]text size 18.75 pixels[[/size]]
+
+First paragraph. Lorem ipsum dolor sit amet, consectetuer adipiscing elit.
+
+Second paragraph. Aenean a libero. Vestibulum adipiscing, felis ac faucibus imperdiet, erat lacus accumsan neque, vitae nonummy lorem pede ac elit.
+Just a new line.
+Another new line.
+
+Only 5x space+ Enter:
+
+Last line after 5 x enter 
+
+With special Line-Break: _
+ _
+ _
+ _
+ _
+Last line after 5 x special line-Break
+
+``quotation''
+`quotation'
+,,quotation''
+<<quotation>>
+>>quotation<<
+dots...
+em -- dash
+
+This //text// gets **parsed**.
+@@This //text// does not get **parsed**.@@
+
+HTML entities: @<U umlaut: &#252;>@
+@<[[code]]>@
+@<Hello world @@ !!!!>@
+@<Something **not** bold>@
+@<[[module ListPages]]>@
+@<Copyright sign: &copy;>@
+@<[[/code]]>@
+Or, @<@&lt;>@ and @<&gt;@>@
+
+[!--
+This text will not be rendered.
+--]
+[!--This text will not be rendered.--]
+
++ Level 1 Heading
+++ Level 2 Heading
++++ Level 3 Heading
+++++ Level 4 Heading
++++++ Level 5 Heading
+++++++ Level 6 Heading
+
++ This section appears in the TOC
++* And this one does not
+++* Neither does this one
```

Comment:
wtf this is a wikidot feature?

I don't see anything about it in https://www.wikidot.com/doc-wiki-syntax:headings

Reply:
Well, this absolutely is copied from Wikidot. Let me see if I can find it...

https://www.wikidot.com/doc-wiki-syntax:table-of-contents